app/views/components/_autocomplete.html.erb

```diff
@@ -44,7 +44,7 @@
         multiple: multiple
       %>
     <% else %>
-      <%= tag.p "No options available", class: "govuk-hint" %>
+      <%= tag.p "No options available", class: "govuk-hint" unless type=='search-topics' %>
```

Comment:
Minor, `type == 'search-topics' `

Reply:
Ah, that bit of code is outdated. We're only showing this message for type `multiple`, which is more adequate.